nipoppy/cli/parser.py

```diff
@@ -188,6 +189,33 @@ def add_subparser_init(
     return parser
 
 
+def add_subparser_status(
+    subparsers: _SubParsersAction,
+    formatter_class: type[HelpFormatter] = HelpFormatter,
+) -> ArgumentParser:
+    """Add subparser for status command."""
+    description = "Show current status of the dataset."
+    parser = subparsers.add_parser(
+        COMMAND_STATUS,
+        description=description,
+        help=description,
+        formatter_class=formatter_class,
+        add_help=False,
+    )
+
+    parser = add_arg_dataset_root(parser)
+
+    # TODO in future release
+    # parser.add_argument(
+    #     "--save-status-to-disk",
+    #     action="store_true",
+    #     help=("Save the status to disk."
+    #           " This will overwrite the existing status file."),
+    # )
```

Comment:
I am fine with commenting these out but my top preference would just be to delete these lines and open another issue for a future PR...

Reply:
okay i will remove it